rust-src/id/src/identity_provider.rs

```diff
@@ -443,30 +441,41 @@ fn compute_message<P: Pairing, AttributeType: Attribute<P::ScalarField>>(
         return Err(Reason::TooManyAttributes);
     }
 
+    let mut gs = Vec::new();
+    let mut exps = Vec::new();
```

Comment:
Performance wise it really won't matter, but I think we should use `with_capacity` because there is no reason not to.